docs/architecture/resilience.txt

```diff
@@ -290,6 +299,54 @@ matches the lost update. This will break `Optimistic Concurrency Control
 <https://crate.io/docs/reference/sql/occ.html>`_.
 
 
+Replicas can fall out of sync when a primary shard fails
+--------------------------------------------------------
+
+.. raw:: html
+
+   <table class="summary">
+   <tr>
+    <td>Status</td>
+    <td>
+        Initial work to address this issue has been done by introducing
+        sequence numbers:
+        <a href="https://github.com/elastic/elasticsearch/issues/10708">#10708</a>
+    </td>
+   </tr>
+   <tr>
+    <td>Severity</td>
+    <td>Modest</td>
+   </tr>
+   <tr>
+    <td>Likelihood</td>
+    <td>Rare</td>
+   </tr>
+   <tr>
+    <td>Cause</td>
+    <td>Primary fails and writes only to a subset of its replicas</td>
```

Comment:
would rephrase to: The primary shards fails while in-flight writes are not yet written to all replica shards.

Reply:
+1